api/pkg/apis/v1alpha1/providers/target/k8s/k8s.go

```diff
@@ -630,15 +630,15 @@ func (i *K8sTargetProvider) GetValidationRule(ctx context.Context) model.Validat
 			RequiredMetadata:      []string{},
 			OptionalMetadata:      []string{},
 			ChangeDetectionProperties: []model.PropertyDesc{
-				{Name: model.ContainerImage, IgnoreCase: true, SkipIfMissing: false},
+				{Name: "container.*", IgnoreCase: true, SkipIfMissing: false},
```

Comment:
Please consider to include a test for this change.

Reply:
Manual tested. Do we need an integration for this minor change?

Reply:
Yes. My concern is someone change some logic in the future and you cannot know this feature is broken if we don't have any gates for this scenario.

Reply:
Agree with test cases.

Reply:
Test added and passed.